src/components/Carousel/Carousel.js

```diff
@@ -55,6 +55,7 @@ const GalleryCarousel = (props, ref) => {
     index: props.index,
     isLoop: props.isLoop
   });
+  const thumbnailElements = props.thumbnails || slidesElements;
```

Comment:
`slideElements` sounds better than `slidesElements`. I will rename `slidesElements` after your changes. You do not need to change it!

Reply:
Please add `thumbnails: PropTypes.arrayOf(PropTypes.node),` to `src/components/Carousel/props.js`.